source/runtime.cpp

```diff
@@ -4350,7 +4350,7 @@ template <> void reshade::runtime::get_uniform_value<uint32_t>(const uniform &va
 void reshade::runtime::set_uniform_value_data(uniform &variable, const uint8_t *data, size_t size, size_t base_index)
 {
 #if RESHADE_ADDON
-	if (has_addon_event<addon_event::reshade_set_uniform_value>() && !is_loading() && !_is_in_api_call)
```

Comment:
I think it seems not have effect.
I thought the has_addon_event should use if call add-on with high cost processing(e.g. make big array, call syscall, ...)
Inserted at https://github.com/crosire/reshade/blame/10e449d7e71aab07fe7b51a4abc7ea6028f7f6be/source/runtime.cpp#L4295-L4304

Reply:
Yes, that is true. I only did this change for code aesthetics, there is no other practical reason behind it